src/crimson/common/logfile.cc

```diff
@@ -0,0 +1,76 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include <fstream>
+#include <seastar/util/log.hh>
+
+#include "common/debug.h"
+#include "logfile.h"
+#include "log.h"
+
+#define dout_context g_ceph_context
+#define dout_prefix *_dout
+#define dout_subsys ceph_subsys_
+
+namespace {
+  seastar::logger& logger() {
+    return crimson::get_logger(ceph_subsys_osd);
```

Comment:
is the logging subsystem used here? actually, we are supposed to set up the infra for the logging subsystem here.

Reply:
i thought that i can set the ostream in the function, need to remove it

Reply:
Isn't it a unique file name for each osd? or I'm missing something here?

Reply:
sorry for the confusion, i am trying to understand the reason why you are setting up `dout_context`, `dout_prefix` and `dout_subsys`.